src/dotnet/Mgmt.CI.BuildTools/NugetToolsPackage/CI.Tools.Package/build/targets/common.Build.props

```diff
@@ -0,0 +1,37 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<PropertyGroup>
+		<Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+		<!--<GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+		<IncludeSymbols>true</IncludeSymbols>-->
+		<!-- <RestorePackagesPath>$(LibraryNugetPackageFolder)</RestorePackagesPath>
+		<NugetCommonProfileTags></NugetCommonProfileTags>
+		<PackageOutputPath>$(pkg_root_RepoNugPkgOutputDir)</PackageOutputPath>
+		<AddProjectReferenceForDebuggingPurpose>false</AddProjectReferenceForDebuggingPurpose>
+		<AddNugetReferenceForCIandCmdlineBuild>true</AddNugetReferenceForCIandCmdlineBuild>
+		<SkipBuildingTestProject>false</SkipBuildingTestProject>
+		<SignAssembly>true</SignAssembly>
```

Comment:
Why are these properties commented out? They broken the signing for the libraries using these tools.